solr/core/src/java/org/apache/solr/api/V2HttpCall.java

```diff
@@ -360,6 +366,80 @@ protected void execute(SolrQueryResponse rsp) {
     SolrCore.postDecorateResponse(handler, solrReq, rsp);
   }
 
+  @Override
+  protected void populateTracingSpan(Span span) {
+    // Set db.instance
+    String coreOrColName = this.origCorename;
+    if (coreOrColName == null) {
+      Map<String, String> pathTemplateValues = getUrlParts(); // == solrReq.getPathTemplateValues()
+      coreOrColName = pathTemplateValues.get("collection");
+      if (coreOrColName == null) {
+        coreOrColName = pathTemplateValues.get("core");
+      }
+    }
+    if (coreOrColName != null) {
+      span.setTag(Tags.DB_INSTANCE, coreOrColName);
+    }
+
+    // Get the templatize-ed path
+    String path;
+    if (api instanceof AnnotatedApi) {
+      // ideal scenario; eventually everything might be AnnotatedApi?
+      path = ((AnnotatedApi) api).getEndPoint().path()[0]; // consider first to be primary
+    } else {
+      path = computeEndpointPath();
+      // TODO consider getValidators, looking for command & path?
+    }
+
+    String verb = req.getMethod().toLowerCase(Locale.ROOT);
+    // if this api has commands ...
+    final Map<String, JsonSchemaValidator> validators = getValidators(); // should be cached
+    if (validators != null && validators.isEmpty() == false && solrReq != null) {
+      // does this request have one command?
+      List<CommandOperation> cmds = solrReq.getCommands(false);
```

Comment:
@noblepaul I'm unsure if `getCommands(false)` should be that or `true` -- a "validateInput" argument.  I was thinking not validate to save CPU for this use case but getCommands caches its result, and thus I think future calls to getCommands(true) will return cached results of non-validated commands.  And that will happen typically.  I'll change this back to `true`.

Reply:
I would use `false`